interactions/api/models/guild.py

```diff
@@ -768,6 +768,27 @@ async def create_channel(
 
         return Channel(**res, _client=self._client)
 
+    async def clone_channel(self, channel_id: int) -> Channel:
+        """
+        Clones a channel of the guild.
+
+        :param channel_id: The id of the channel to clone
+        :type channel_id: int
+        """
+        if not self._client:
+            raise AttributeError("HTTPClient not found!")
+        ch = Channel(**await self._client.get_channel(channel_id=channel_id))
+
+        ch._json["permission_overwrites"] = [
+            Overwrite(**_) for _ in ch._json["permission_overwrites"]
+        ]
+        [
+            ch._json.pop(attr, None)
```

Comment:
I just noticed, this is senseless. You can just modify the value from `get_channel`, you don't need to create an object from it

Reply:
WDYM?